mantidproject/mantid
# script editor output window to match zoom with window

zooming in the output and script parts of the script editor should now be linked

fixes #9566
### Release notes

Added
- The text size is now linked between the script and output parts of the window, so if you zoom into or out of one the other follows.  You can zoom using ctrl+the mouse scroll wheel, or the font size options under the window menu.
### to test
1. open the script editor with at least two tabs both with some code (print statements) and output
2. zoom 
   1. with the mouse wheel both on the script and output windows
   2. with the menu options
   3. with the reset zoom menu option
3. check that the zoom is linked, both within the tab and across all tabs
